### PR TITLE
Sync with internal repo

### PR DESCRIPTION
**Pull request by m-rudyk**
# Description

This pull request aims to create a reusable package, promoting code modularity and easier integration across projects.
Synchronize open-source and internal LPVS versions, ensuring consistency and improved collaboration between the two environments.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [x] Test Coverage update

# How Has This Been Tested?
Deploy and test localy. 

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by tiokim** [comment]
Thanks for opening this PR. Can you break this down in multiple PRs for reviewing and approving them easily?

File: src/main/java/com/lpvs/service/LPVSGitHubService.java [hunk 1]
```diff
@@ -248,19 +251,24 @@ public void commentResults(LPVSQueue webhookConfig, List<LPVSFile> scanResults,
...
+                commitCommentBuilder.append("</ul>");
+                commitComment += commitCommentBuilder.toString();
             }
 
             if (hasProhibitedOrRestricted || hasConflicts) {
                 lpvsPullRequest.setStatus(LPVSPullRequestStatus.ISSUES_DETECTED.toString());
                 pullRequestRepository.save(lpvsPullRequest);
-                pullRequest.comment("**\\[License Pre-Validation Service\\]** Potential license problem(s) detected \n\n" + commitComment);
+                pullRequest.comment("**\\[License Pre-Validation Service\\]** Potential license problem(s) detected \n\n" +
+                        commitComment + "\n\n###### <p align='right'>Check the validation details on [SOSHUB LPVS](" +
```
Comment (o-kopysov) [inline src/main/java/com/lpvs/service/LPVSGitHubService.java:262]: SOSHUB is an internal system and should be deleted from this repo.

File: src/main/java/com/lpvs/service/LPVSGitHubService.java [hunk 2]
```diff
@@ -248,19 +251,24 @@ public void commentResults(LPVSQueue webhookConfig, List<LPVSFile> scanResults,
...
+                        webhookConfig.getHubLink() + ")</p>");
                 repository.createCommitStatus(webhookConfig.getHeadCommitSHA(), GHCommitState.FAILURE, null,
                         "Potential license problem(s) detected", "[License Pre-Validation Service]");
             } else {
                 lpvsPullRequest.setStatus(LPVSPullRequestStatus.COMPLETED.toString());
                 pullRequestRepository.save(lpvsPullRequest);
-                pullRequest.comment("**\\[License Pre-Validation Service\\]**  No license issue detected \n\n" + commitComment);
+                pullRequest.comment("**\\[License Pre-Validation Service\\]**  No license issue detected \n\n" +
+                        commitComment + "\n\n###### <p align='right'>Check the validation details on [SOSHUB LPVS](" +
```
Comment (o-kopysov) [inline src/main/java/com/lpvs/service/LPVSGitHubService.java:270]: SOSHUB is an internal system and should be deleted from this repo.

**Review by o-kopysov — APPROVED** [review]
LGTM

File: src/main/java/com/lpvs/entity/enums/LPVSPullRequestAction.java [hunk 3]
```diff
@@ -12,7 +12,9 @@ public enum LPVSPullRequestAction {
     REOPEN("reopened"),
     CLOSE("closed"),
     UPDATE("synchronize"),
-    RESCAN("rescan");
+    RESCAN("rescan"),
+    REVIEW_BOT_SCAN("review_bot_scan"),
+    FAST_SCAN("fast_scan");
```
Comment (tiokim) [inline src/main/java/com/lpvs/entity/enums/LPVSPullRequestAction.java:17]: I don't know why we need these enums.

File: src/main/java/com/lpvs/service/LPVSGitHubService.java [hunk 4]
```diff
@@ -302,7 +308,7 @@ public String getRepositoryLicense(LPVSQueue webhookConfig) {
 
 
     public void setGithubTokenFromEnv() {
-        if (System.getenv("LPVS_GITHUB_TOKEN") != null) GITHUB_AUTH_TOKEN = System.getenv("LPVS_GITHUB_TOKEN");
+            if (System.getenv("LPVS_GITHUB_TOKEN") != null) GITHUB_AUTH_TOKEN = System.getenv("LPVS_GITHUB_TOKEN");
```
Comment (tiokim) [inline src/main/java/com/lpvs/service/LPVSGitHubService.java:311]: Please check the indentation.

**Review by tiokim — CHANGES_REQUESTED** [review]
PTAL

**Review by v-kerimov — APPROVED** [review]

File: src/main/java/com/lpvs/entity/enums/LPVSPullRequestAction.java (hunk 3 above)
Comment (m-rudyk) [reply src/main/java/com/lpvs/entity/enums/LPVSPullRequestAction.java]: you are right, thank you.

**Review by tiokim — APPROVED** [review]
LGTM!

**Pull request merged**